microsetta_interface/tests/test_integration.py

```diff
@@ -0,0 +1,161 @@
+import unittest
+import hashlib
+import os
+import uuid
+from unittest.mock import patch
+import jwt
+from microsetta_interface.server import app
+from microsetta_interface.implementation import TOKEN_KEY_NAME
+from microsetta_interface.config_manager import SERVER_CONFIG
+import requests
+
+
+# check if microsetta-private-api appears to be running
+PRIVATE_API_AVAILABLE = False
+try:
+    req = requests.get(SERVER_CONFIG['private_api_endpoint'] + '/ui')
+except:  # noqa
+    PRIVATE_API_AVAILABLE = False
+else:
+    PRIVATE_API_AVAILABLE = (req.status_code == 200)
+
+
+# obtain the JWT key if it exists
+PRIVKEY_ENVVAR = 'MICROSETTA_INTERFACE_DEBUG_JWT_PRIV'
+if os.environ.get(PRIVKEY_ENVVAR, False):
+    PRIV_KEY = open(os.environ[PRIVKEY_ENVVAR]).read()
+else:
+    PRIVATE_API_AVAILABLE = False
+
+
+def _fake_jwt(email, verified):
+    if PRIVATE_API_AVAILABLE:
+        # lie and say we're from authrocket
+        payload = {'email': email,
+                   'email_verified': verified,
+                   'iss': 'https://authrocket.com',
+                   'sub': hashlib.md5(email.encode('ascii')).hexdigest()}
+        encoded = jwt.encode(payload, PRIV_KEY, algorithm='RS256')
```

Comment:
This seems overcomplicated.  Recommend checking out https://jwt.io/  You can build a fake token with base64, no private key file required.  Afaik, you can specify the sub as anything you like, all that matters is there's an entry for it in the database.

Reply:
Oh that's nifty, thanks, and it would be great to not generate keys and stuff

sub is used on legacy account match in -private-api so has to be something we control, and specific to the user, but open ended otherwise

Reply:
Either there's something I completely misunderstand, or we cannot get away from using keys. Per jwt.io, the RS256 algorithm requires the use of a public and private key. Our verify in code only supports RS256. 

Generating keys is annoying but not complex, and currently works, so chalking this up to a possible change to consider in the future but not doing it here